### DISCUSSION TITLE
What is wrong with my tooltip setup? Button press events aren't working when a tooltip is displayed...

### DISCUSSION THREAD

**Discussion by e-cooper** [adobe/react-spectrum#2202] [Q&A] [answered]
[Here's a codesandbox](https://codesandbox.io/s/festive-ives-wiqxu) that demonstrates the issue. I've also included the Spectrum implementation which is set up similarly, but works fine.

Any help is appreciated 🙏

**Comment by snowystinger** [comment, marked as answer]
~I think you just need to merge otherProps with the rest. I think right now your handler is being overwritten~ sorry was on my phone earlier, i tried my suggestion and it did not work
What are some differences you have from our implementation? I'd start changing it to be more like ours and see at what point it starts working. I don't see anything obvious yet.

**Reply by e-cooper** [reply]
Thanks for the reply. I kept swapping out components and finally narrowed it down to an issue in my `PopoverContainer` component. Turns out, we need to provide `isDismissable: false` to the `useModal` hook in this case.

**Reply by snowystinger** [reply]
Glad you were able to figure it out! Just to clarify for anyone else coming through here, you meant
> Turns out, we need to provide isDismissable: false to the useModal hook in this case

useModal => useOverlay

interestingly, I tried making that false in your codesandbox and couldn't reproduce the issue, so I think there was something else from the original
